src/Chapter13/Table13.01.d.LambdaExpressionNotesAndExamples.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Justification: Only snippets of source code shown for elucidation.
+#pragma warning disable CS0168 // Variable is declared but never used
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter13.Table13_01;
+
+public partial class LambdaExpressionNotesAndExamples
+{
+    // 4.
+    static public void MemberMethodsOnExpressions()
+    {
+    //#if COMPILEERROR
```

Comment:
Do we need these commented lines?

Reply:
Keeping these here for continuity in the fact that these exist in the text.